apps/system/test/unit/rocketbar_test.js

```diff
@@ -666,6 +666,13 @@ suite('system/Rocketbar', function() {
     test('Search window without front window', function() {
       var respond =
         subject.respondToHierarchyEvent(new CustomEvent('system-resize'));
+      assert.isFalse(respond);
+    });
+
+    test('Rocketbar is inactive', function() {
+      subject.deactivate();
+      var respond =
```

Comment:
nit: for all those tests let's rename `respond` to `stopsPropagation`, it makes the tests more readable / document what we're actually asserting.